sdk/storage/Azure.Storage.DataMovement/src/Shared/DataMovementConstants.cs

```diff
@@ -96,118 +101,126 @@ internal static class PlanFile
             /// <summary>Index: 84</summary>
             internal const int PartNumberIndex = TransferIdIndex + TransferIdStrNumBytes;
             /// <summary>Index: 92</summary>
-            internal const int SourcePathLengthIndex = PartNumberIndex + LongSizeInBytes;
+            internal const int SourceResourceIdLengthIndex = PartNumberIndex + LongSizeInBytes;
             /// <summary>Index: 94</summary>
+            internal const int SourceResourceIdIndex = SourceResourceIdLengthIndex + UShortSizeInBytes;
+            /// <summary>Index: 114</summary>
+            internal const int SourcePathLengthIndex = SourceResourceIdIndex + ResourceIdMaxStrLength;
+            /// <summary>Index: 116</summary>
             internal const int SourcePathIndex = SourcePathLengthIndex + UShortSizeInBytes;
-            /// <summary>Index: 8,286</summary>
+            /// <summary>Index: 8308</summary>
             internal const int SourceExtraQueryLengthIndex = SourcePathIndex + PathStrNumBytes;
-            /// <summary>Index: 8,288</summary>
+            /// <summary>Index: 8310</summary>
             internal const int SourceExtraQueryIndex = SourceExtraQueryLengthIndex + UShortSizeInBytes;
-            /// <summary>Index: 10,288</summary>
-            internal const int DestinationPathLengthIndex = SourceExtraQueryIndex + ExtraQueryNumBytes;
-            /// <summary>Index: 10,290</summary>
+            /// <summary>Index: 8312</summary>
+            internal const int DestinationResourceIdLengthIndex = SourceExtraQueryIndex + UShortSizeInBytes;
```

Comment:
this needs to be 
"DestinationResourceIdLengthIndex = SourceExtraQueryIndex + ExtraQueryNumBytes;"

Because the value above it is an extra query